front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -131,6 +131,11 @@ const TimetableTrainCard = ({
     dispatch(updateTrainIdUsedForProjection(train.id));
   };
 
+  const isAfterMidnight = dayjs(train.arrivalTime, 'D/MM/YYYY').isAfter(
```

Comment:
Could we maybe copy over the `/* TODO: delete the format when the date management `PR` has been passed */` just so that we don't forget about this line?